### DISCUSSION TITLE
HELP, 怎么使用antd代码下面附加的css?

### DISCUSSION THREAD

**Discussion by mystery-treasure** [ant-design/ant-design#35169] [Q&A]
我是个初学者，最近发现一些需要copy的代码下面的虚线下都有一个css样式，比如Layout
我不知道怎么使用它，谁能麻烦给个例子啊？
我把它复制到index.css里了，是下面的样子，但是运行的时候就是没有加载到css.
我需要怎么做呢？
@import '~antd/dist/antd.css';
.logo {
  float: left;
  width: 120px;
  height: 31px;
  margin: 16px 24px 16px 0;
  background: rgba(255, 255, 255, 0.3);
}
.ant-row-rtl .logo {
  float: right;
  margin: 16px 0 16px 24px;
}
.site-layout-background {
  background: #fff;
}

**Comment by yoyo837** [comment]
不建议初学者直接接触库或者框架，无论是学习还是工作。

**Reply by mystery-treasure** [reply]
谢谢你的回答，不过，针对我提出的问题可以做些帮助吗？

**Reply by mystery-treasure** [reply]
Soved.
antd升级到4.20问题解决
